server/main.go

```diff
@@ -12,11 +12,9 @@ import (
 	"strconv"
 	"time"
 )
-
 const recordCount = 1000000
 const columnCount = 11
 const delayResponse = 500 * time.Millisecond
-
 var columns = [columnCount]string{"ID", "A", "B", "C", "D", "E", "F", "G", "H", "I", "J"}
```

Comment:
There should be any changes to this file. Looks like you accidentally removed the new lines. Make sure this file doesn't even popup on Files changed on GitHub.